initfiles/componentfiles/configxml/esp.xsl

```diff
@@ -47,9 +47,16 @@
    </xsl:variable>
 
     <xsl:variable name="espProcess" select="/Environment/Software/EspProcess[@name=$process]"/>
+    <xsl:variable name="espBindingProtocol">
+        <xsl:choose>
+            <xsl:when test="/Environment/Software/EspProcess[@name=$process]/EspBinding[@protocol='https']">https</xsl:when>
+            <xsl:otherwise>http</xsl:otherwise>
```

Comment:
If we ever have configurations that use other protocols this could be a problem (For example some ESPs in LN use EDITS).  Perhaps for now the logic should be... if you can find https use it, if you can find http use it, otherwise, no control port.  Then we could add control for other protocols if we wanted to later:

```
<xsl:choose>
   <xsl:when test="/Environment/Software/EspProcess[@name=$process]/EspBinding[@protocol='https']">https</xsl:when>
   <xsl:when test="/Environment/Software/EspProcess[@name=$process]/EspBinding[@protocol='http']">http</xsl:when>
   <xsl:otherwise>none</xsl:otherwise>
```
Then, check for none below and don't add a control part at all.